### ISSUE TITLE
Is this still working?

### ISSUE THREAD

**Issue by hmaesta** [beyonk-adventures/svelte-facebook-pixel#4]
I can't make it work on a clean REPL. [See it here.](https://svelte.dev/repl/6fe03e84f41b42a188b3f45da2cf3d25?version=3.35.0)

Console returns `_fbq is not a function`.

.

I realised that [svelte-google-analytics](https://github.com/beyonk-adventures/svelte-google-analytics) has a better approach using [async-script-loader](https://github.com/beyonk-adventures/async-script-loader) and tried to "replace" Google Analytics by Facebook Pixel code, but I freeze in the `callback()` part because Facebook's original code is way too obfuscated.

**Comment by antony** [comment]
I think you're getting the issue that has already been raised here: https://github.com/beyonk-adventures/svelte-facebook-pixel/issues/2

However, we use this in production so I'm a bit confused as to why it wouldn't work.

I'll have a poke around soon and see if they've updated their API or something and broken it. Facebook are abysmal at API versioning, or APIs in general.

**Comment by benwoodward** [comment]
I found that I needed to use `fb.enable()` for it to work, because by default it doesn't initialise itself. However, I am getting the same issue mentioned in #2 about Duplicate Pixel ID. Have only tested in dev so far, will update if problem persists in production.

EDIT: I'm guessing that it's due to this line: https://github.com/beyonk-adventures/svelte-facebook-pixel/blob/2b33a1eb3b137df36d807cc5678f59a9b5029c83/src/FacebookPixel.svelte#L49

If you call `fb.enable()` that in turns calls `mount()`, and `init()`, and `mount()` calls `enable()` on the line linked above, which means that `init()` ends up getting called twice if you call `enable()` .. I think

EDIT: Made a PR to fix the issue: https://github.com/beyonk-adventures/svelte-facebook-pixel/pull/5
